testRearrangeCarGroup8.py

```diff
@@ -0,0 +1,72 @@
+import logging
+import unittest
+import numpy as np
+from RearrangeCarGroup8Movements import RearrangeCars
+
+class TestRearrangeCars(unittest.TestCase):
+
+    '''Unittests for RearrangeCars class and promised fuctionality
+    '''
+
+    def setUp(self):
+
+        '''Basic setting that will be used in several tests
+        '''
+        self.init_order = [1, 2, 0, 3]
+        self.final_order = [3, 1, 2, 0]
+        self.rc = RearrangeCars(self.init_order, self.final_order)
+
+    def test_rearrange_all_cars(self):

```

Comment:
Maybe rearrange_sample_cars? (Minor change, but you know immediately what's up :D)